chef_master/source/dk_windows.rst

```diff
@@ -1,101 +1,195 @@
+   chef-client version: 14.4.56
+   berks version: 7.0.6
+   kitchen version: 1.23.2
+   inspec version: 2.2.70
+
+Configure the Windows Environment
 -----------------------------------------------------
-.. tag ruby_set_system_ruby_as_chefdk_ruby_windows_user_profile
 
-On new Windows machines, commands cannot be appended to ``$PROFILE`` if the folder does not exist, or if there is a new user profile. This will result in an error similar to the following:
+Use the ``chef shell-init`` command to configure your environment to use the ChefDK locations for Chef and Ruby. Set the execution policy, create a profile, and then create the initialization script for your environment with ``chef shell-init``.
+
+Set the Execution Policy
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+On newer Windows machines, PowerShell scripts will not work until an administrator runs the following command:
 
 .. code-block:: bash
 
-   PS C:\Users\Stuart> "chef shell-init powershell | Invoke-Expression" >> $PROFILE
-   out-file : Could not find a part of the path
-   'C:\Users\Stuart\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1'.
-   At line:1 char:1
-   + "chef shell-init powershell | Invoke-Expression" >> $PROFILE
-   + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-       + CategoryInfo          : OpenError: (:) [Out-File], DirectoryNotFoundException
-       + FullyQualifiedErrorId : FileOpenFailure,Microsoft.PowerShell.Commands.OutFileCommand
+   Set-ExecutionPolicy RemoteSigned
 
-In this situation, run the following ``chef shell-init`` command instead:
+Create a Profile
++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-.. code-block:: bash
+Once the execution policy is in place, initialize the Windows environment with the following commands: 
 
-   if(Test-Path $PROFILE){ chef shell-init powershell | Add-Content $PROFILE } else { New-Item -Force -ItemType File $PROFILE; chef shell-init powershell | Add-Content $PROFILE }
+Create a ``$PROFILE`` file if one doesn’t exist already:
 
-.. end_tag
+.. code-block:: shell
 
-Set the Execution Policy
------------------------------------------------------
-.. tag ruby_set_system_ruby_as_chefdk_ruby_windows_admin
+   PS C:\Users\<username>> if(!(Test-Path $PROFILE)) { New-Item -Force -ItemType File $PROFILE }
 
-On new Windows machines, PowerShell scripts will not work until an administrator runs the following command:
+Next, add the output of ``chef shell-init powershell`` to your ``$PROFILE`` (This command does not print the output onto the screen):
 
-.. code-block:: bash
+.. code-block:: shell
 
-   Set-ExecutionPolicy RemoteSigned
+   PS C:\Users\<username>> chef shell-init powershell | Add-Content $PROFILE
 
-.. end_tag
+Once initialized, your paths and environment variables are set up correctly for every new PowerShell session. See your environment settings by using ``cat $PROFILE``:
+
+.. code-block:: shell
+
+    PS C:\Users|<username>> cat $PROFILE
```

Comment:
cat is OK but not a native command - as some PowerShell aliases are going away, the canonical way to write this without aliases
```
Get-Content $PROFILE
```

Reply:
done